app/controllers/hathi_controller.rb

```diff
@@ -1,9 +1,31 @@
 class HathiController < ApplicationController
 
+  def hathi_access
+    return hathi_access_bib_status if params[:bib_id]
+    return hathi_access_oclc_status if params[:oclc]
+  end
+
   def hathi_access_bib_status
-    if params[:bib_id]
-      @hathi_record = HathiAccess.where(bibid: params[:bib_id]).select(:bibid,:status,:origin,:oclc_number)
-      render json: @hathi_record
+    @record = HathiAccess.where(bibid: params[:bib_id])
+    if @record.present?
+      render json: @record, except: [:id, :created_at, :updated_at]
+    else
+      status_404
```

Comment:
I kept the bib_id lookup in case we decide that we want to use it in the near future.